packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -260,6 +264,16 @@ export function arrayToProtoPODGPCCircuitParam(
   );
 }
 
+/**
+ * Mapping computing the maximum number of virtual entries from given GPC
+ * parameters.
+ */
+export function paramMaxVirtualEntries<
+  GPCCircuitConfig extends ProtoPODGPCCircuitParams
```

Comment:
Do you need the extends here, vs. using ProtoPODGPCCircuitParams directly?
I'd expect TypeScript to simply allow the object to be passed as this "superclass".

Reply:
Seems not.